meganav.yaml

```
@@ -1,3 +1,699 @@
+              description: Open source software storage
+              url: /ceph
+            - title: Anbox cloud
+              description: Virtual Android devices on demand
+              url: https://anbox-cloud.io/
+            - title: Open Source MANO
+              description: Network Functions Virtualisation
+              url: /telco/osm
+            - title: JAAS
+              description: Juju as a Service
+              url: https://jaas.ai/
+            - title: Private cloud build
+              description: OpenStack implementation
+              url: /openstack/consulting
+      secondary_links:
+        - title: Links
+          links:
+            - title: Server documentation
+              url: /server/docs
+            - title: MAAS documentation
+              url: https://maas.io/docs
+            - title: MicroStack installation
+              url: https://microstack.run/docs/single-node
+            - title: Data Centre Networking blog
+              url: /blog/data-centre-networking-what-is-sdn
+            - title: OpenStack pricing
+              url: /openstack/pricing-calculator
+            - title: Private cloud consulting
+              url: /openstack/consulting
+            - title: "Private cloud: Avoiding the high cost of operations"
+              url: /blog/private_cloud_avoiding_operational_costs
+
+    - title: Public cloud
+      primary_links:
+        - title: Products
+          links:
+            - title: Ubuntu on AWS
+              description: AWS Marketplace
+              url: /aws
+            - title: Ubuntu on Azure
+              description: Microsoft Azure Marketplace
+              url: /azure
+            - title: Ubuntu on Google Cloud
+              url: /gcp
+
+    - title: Virtualisation
+      primary_links:
+        - title: Products
+          links:
+            - title: Multipass
+              description: Ubuntu VMs for Windows, macOS & Linux
+              url: https://multipass.run/
+            - title: LXD
+              description: Linux containers
+              url: https://linuxcontainers.org/
+            - title: WSL
+              description: Windows subsytem for Linux
+              url: /wsl
+
+      secondary_links:
+        - title: Links
+          links:
+            - title: What is Virtualisation?
+              url: https://snapcraft.io/blog/what-is-virtualisation-the-basics
+            - title: Multipass Documentation
+              url: https://multipass.run/docs
+            - title: Virtualization docs in the server guide
+              url: /server/docs/virtualization-multipass
+            - title: What is WSL?
+              url: https://wiki.ubuntu.com/WSL
+            - title: Install Ubuntu on Windows 10
+              url: /tutorials/install-ubuntu-on-wsl2-on-windows-10
+            - title: Install Ubuntu on Windows 11
+              url: /tutorials/install-ubuntu-on-wsl2-on-windows-11-with-gui-support#1-overview
```

```suggestion
              url: /tutorials/install-ubuntu-on-wsl2-on-windows-11-with-gui-support
```
We should either remove the '#' here, or add it in the other cases. Behavior is the same